evm_loader/Cargo.toml

```diff
@@ -10,3 +10,13 @@ members = [
     'program',
     'program-macro',
 ]
+
+[workspace.dependencies]
+solana-clap-utils = "=1.17.28"
```

Comment:
The PR misses changes in Cargo.lock file

Reply:
Fixed.